README.md

```diff
@@ -5,26 +5,44 @@
 👋 Welcome to **Matchstick** - a unit testing framework for The Graph protocol. Try out your mapping logic in a sandboxed environment and ensure your handlers run correctly when deploying your awesome subgraph!
 
 ## Quick Start 🚀
-The release binary comes in three flavours - for **macOS**, **Linux** and **Windows**. To add **Matchstick** to your subgraph project just open up a terminal, navigate to the root folder of your project and simply run `graph test` - it downloads the latest Matchstick binary and runs the specified test or all tests in a test folder (or all existing tests if no datasource flag is specified). Example usage: `graph test Gravity`.
+### Docker 🐳
+The quickest way to use **Matchstick** "out of the box" is to build and run an ubuntu-based Docker container with a Matchstick image. In order to do that you will need to have [Docker](https://docs.docker.com/get-docker/) installed on your machine. 
 
-❗ If you don't have Postgres installed, you will need to install it. Instructions for that below:
+Next, create a file named Dockerfile in the root folder of your subgraph project, and paste [the contents of this file](https://github.com/LimeChain/demo-subgraph/blob/main/Dockerfile) there.
+
+Now you can build a Matchstick image using the following command:
+```
+docker build -t matchstick .
+```
+
+The build step might take a while, but once that's done we can quickly run our tests like this:
+```
+docker run --rm matchstick
+```
+
+If you want to pass in arguments to **Matchstick** (for instance to test only a specific datasource or to run the test coverage tool) you can do so like this:
+```
+docker run -e ARGS="gravity" --rm matchstick
+```
```

Comment:
I think it will be quite cool to abstract all this in an executable shell script and invoke it like the regular matchstick cli.

Reply:
But this may be added as another issue, of course.

Reply:
Absolutely, all this could be bundled in a matchstick-docker.sh or something like that